modules/managing-feature-flags.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * configuration/managing-preview-features.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="managing-feature-flags_{context}"]
+= Managing Feature Flags
+
+== Prerequisites
+* You have access to the environment where the {product-title-short} component is deployed.
+* You have permission to modify environment variables.
+* You understand that the Technological Preview features might be incomplete and are provided with limited support.
+* You know if the feature flag needs to be configured before the deployment. Check the installation manifests to see if they use the required flag.
```

Comment:
What do you mean by this? Are you trying to say that *other* feature flags may need to configured?

Reply:
No, I tried to say that some flags can only be configured during deployment, and have no effect afterward.

Reply:
Maybe "You know if the flag for the Technology Preview feature needs..." to address Stephan's question.